kythe/cxx/indexer/cxx/testdata/objc/generics/generic_bound.m

```diff
@@ -19,10 +19,10 @@ @implementation O2
 // No source range defines BoxDecl since this is a generic type.
 //- @Type defines/binding TypeVar
 //- @Box defines/binding BoxAbs
-//- TypeVar.node/kind absvar
+//- TypeVar.node/kind tvar
 //- BoxDecl childof BoxAbs
```

Comment:
The `BoxDecl childof BoxAbs` assertion is unnecessary but benign; it's probably matching `BoxDecl=doSomething`. `BoxAbs` here and in other tests *could* be renamed, but it's still kind of appropriate ("the box abstraction").

Reply:
Thanks. I'm going to leave them as `BoxAbs` as I'm not sure `BoxIntf` or `BoxThing` is significantly more descriptive.